### ISSUE TITLE
Wrong invert code in Keras Normalization Layer

### ISSUE THREAD

**Issue by heptaflar** [tensorflow/tensorflow#58634]
<details><summary>Click to expand!</summary> 
 
 ### Issue Type

Bug

### Source

source

### Tensorflow Version

tf 2.10

### Custom Code

No

### Python version

3.10

### Current Behaviour?

```shell
The following code is an excerpt from `class Normalization` of `keras.layers.Normalization`.

    def call(self, inputs):
        inputs = self._standardize_inputs(inputs)
        # The base layer automatically casts floating-point inputs, but we
        # explicitly cast here to also allow integer inputs to be passed
        inputs = tf.cast(inputs, self.compute_dtype)
        if self.invert:
            return (inputs + self.mean) * tf.maximum(
                tf.sqrt(self.variance), backend.epsilon()
            )
        else:
            return (inputs - self.mean) / tf.maximum(
                tf.sqrt(self.variance), backend.epsilon()
            )

The code in the `self.invert == True` branch does not invert the code in the `self.invert == False` branch as it shall according to the API documentation, which states that "... layer ... would turn a normalized input back into its original form".
```

### Standalone code to reproduce the issue

```shell
The correct code is:

    def call(self, inputs):
        inputs = self._standardize_inputs(inputs)
        # The base layer automatically casts floating-point inputs, but we
        # explicitly cast here to also allow integer inputs to be passed
        inputs = tf.cast(inputs, self.compute_dtype)
        if self.invert:
            return self.mean + inputs * tf.maximum(
                tf.sqrt(self.variance), backend.epsilon()
            )
        else:
            return (inputs - self.mean) / tf.maximum(
                tf.sqrt(self.variance), backend.epsilon()
            )
```

### Relevant log output

_No response_</details>

**Comment by heptaflar** [comment]
This is my first bug report and I am not sure I have filed all required information correctly, I am afraid. However, the bug is so trivial and obvious to me that I think this might not be necessary.

**Comment by tilakrayal** [comment]
@heptaflar,
Glad the issue is resolved for you, please feel free to move this to closed status. Thank you!

**Comment by heptaflar** [comment]
OK - closed :-)
